src/Symfony/Component/DependencyInjection/Argument/TaggedIteratorArgument.php

```diff
@@ -19,16 +19,33 @@
 class TaggedIteratorArgument extends IteratorArgument
 {
     private $tag;
+    private $indexAttribute;
+    private $defaultIndexMethod;
 
-    public function __construct(string $tag)
+    public function __construct(string $tag, string $indexAttribute = null, string $defaultIndexMethod = null)
```

Comment:
`$indexAttribute` and `$defaultIndexMethod` should get documented. it's not obvious what these params mean.